docs/pages/contributing/documentation.mdx

```diff
@@ -1,5 +1,5 @@
 ---
-title: Contributing to Teleport's Documentation
+title: Teleport documentation resources
```

Comment:
We use title case for titles (sentence case for headings)

Reply:
I prefer the former title. "Documentation resources" could be product docs, and these are docs docs.